packages/space-opera/src/components/reset/reducer.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {reduxStore} from '../../space_opera_base.js';
+
+/** Mostly for unit tests. */
+export const RESET_STATE_ACTION_TYPE = 'RESET_SPACE_OPERA_STATE';
+export function dispatchResetState() {
```

Comment:
Does this really deserve its own file? Seems more like a helper.

Reply:
for consistencies sake, I gave it it's own folder but will probably move it into shared at some point